### ISSUE TITLE
Noob trying to make the Todos example work

### ISSUE THREAD

**Issue by jeromecordiez** [martijnwalraven/meteor-ios#90]
Hey there,

thanks for making this. Finally giving it a spin this weekend, and having no luck so far. Trying to connect to local server from simulator.

The symptoms:
- The Lists list remains desperately empty. Adding a list apparently does nothing.
- Connection is seemingly established OK, despite occasional "Attempting to send message without an open connection'" crashes when debugging.
- Authentication works fine. I can log in and out, and erroneous credentials result in errors, so all good

What I've tried:
- Renaming the subscriptions to lists.public and lists.private, I'm guessing privateLists and publicLists were used before recent Meteor updates
- Forcing the use of 0.2.1

I'll now dive deeper to try to figure out what is going on, but thought I'd post this here already in case there's a proper bug due to recent Meteor updates and it's not just me being stupid :D

**Comment by fxv** [comment]
@jeromecordiez I'm assuming you are using Core Data, so maybe is how you are fetching the data for your tableView. 
**Edit:** I just realized you meant the example project. I recall experiencing some difficulties as well, but using my own project worked fine.

**Comment by jeromecordiez** [comment]
Thanks for your response @fxv, not sure what you mean though?

I basically cloned the Todos example from meteor.com, and the meteor-ios repository.
The web app runs fine (locally).
The iOS app builds, connects to the server, lets me log in and out, but displays nothing nor lets me add items. 

The only thing I changed in the iOS project is the name of the subscriptions, to match the meteor publications.
And I actually today realized I _was_ getting "add" messages from the server, so I'm guessing the problem is somewhere with core data.

**Comment by gg2001** [comment]
Yup. Exact same issue, except I am connected to a heroku server.

**Comment by chenyishun** [comment]
@jeromecordiez 
Has the problem been solved yet? I also encountered the same problem, IOS display panel display blank. Is it because of the latest version of the problem?
I basically cloned the Todos example from meteor.com, and the meteor-ios repository.
The web app runs fine (locally).

**Comment by chrisscholly** [comment]
same issue :( Any update ?

**Comment by brianscroggins24** [comment]
Same issue as well, in one of my own projects I'm subscripting and I can see that the documents are being generated by the METIncrementalStore after inserting some NSLog stagements, but my FetchedResultsController isn't updating at all. It's finding no documents, really at a loss for where to go next, i've dug into the framework and don't know what to do, any help would be appreciated

**Comment by chrisscholly** [comment]
Seems the official Meteor Todos project has changed and is no more compatible with the meteor-ios Todos Example.

To make things work, you can checkout an earlier version of Meteor Todos:
`$ git clone https://github.com/meteor/todos.git`
`$ cd todos`
`$ git checkout 4ad2706f88f7d6ad9059a52fd0a13f3442a57cca`
`$ meteor npm install`
`$ meteor`

Tested and it works as well :)
